### ISSUE TITLE
Page doesn't load at all when using pr.new

### ISSUE THREAD

**Issue by darrensapalo** [stackblitz/webcontainer-core#924]
### Describe the bug

My objective is to be able to load this specific file that I want to edit using Web Publisher. I am following [this article](https://developer.stackblitz.com/codeflow/content-updates-with-web-publisher#edit-this-page-on-github-link) under the section: "Edit this page on Github" link.

I follow the instructions, pick out the file to edit ,and I try to load up the following page:

```
https://pr.new/darrensapalo/fisher-work/blob/master/README.md
```

But nothing loads when the page opens. There are some console errors shown.

### Link to the blitz that caused the error

https://stackblitz.com/~/github.com/darrensapalo/fisher-work/blob/master/README.md

### Steps to reproduce

**Browser:** Brave
**Operating System:** MacOS 12.3.1 (M1 Chip)

1. Go to `https://pr.new/darrensapalo/fisher-work/blob/master/README.md`. This repository is public.

### Expected behavior

The page loads with a user interface for editing the content of that file.

### Parity with Local

- [X] I have [run the project in my local machine](https://github.com/stackblitz/webcontainer-core/blob/main/repro.md#try-locally) and I could not reproduce the issue.

### Screenshots

![image](https://user-images.githubusercontent.com/2420577/208249241-dce34792-93db-47c7-a502-576c47a1200b.png)

### Platform

```
Version = 1.67.0
Hash = 37f869f781e6a426c2a5b3d0f2b304de289c96e9
WebContainer = 6ebe18b1e1db1833797e3e4eb6ff802c375ee61b

Browser name  = Chrome
Full version  = 108.0.0.0
Major version = 108
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 121460244,
  "usedJSHeapSize": 117404280,
  "jsHeapSizeLimit": 2172649472
}
```

**Comment by SamVerschueren** [comment]
Hey @darrensapalo 👋 . Is it possible you are using Brave? Can you check the documentation for Brave here? https://developer.stackblitz.com/platform/webcontainers/browser-config-brave

**Comment by d3lm** [comment]
Hey 👋 Wanna make sure that we have enough information to verify if this is an issue or not. @darrensapalo Can you please provide more info and did you by any chance check our documentation for Brave?

**Comment by darrensapalo** [comment]
Hey guys, thanks for the quick responses. I checked (following the steps to replicate I mentioned above) and it appears to be loading successfully to me now.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/2420577/222955823-f294cc5f-a601-4353-bde6-4b6ed95da1d3.png">

Following the instructions by @SamVerschueren , I checked my brave config settings and my use case still works even if I do not have the cookies enabled specifically for stackblitz.

<img width="720" alt="image" src="https://user-images.githubusercontent.com/2420577/222955885-3ffb4c0a-ce2f-4101-b93c-2bd41d7da9d0.png">

I'm closing this ticket now, as the `https://pr.new/darrensapalo/fisher-work/blob/master/README.md.` link works successfully for me.
